_health-care/_js/components/personal-information/VAInformationSection.jsx

```diff
@@ -17,17 +17,29 @@ class VaInformationSection extends React.Component {
           <ErrorableCheckbox
               label="Are you VA Service Connected 50% to 100% Disabled?"
               checked={this.props.data.isVaServiceConnected}
+              required={!false}
               onValueChange={(update) => {this.props.onStateChange('isVaServiceConnected', update);}}/>
+          {this.props.data.isVaServiceConnected === true &&
+            <ErrorableCheckbox
+                label="Are you compensable VA Service Connected 0% - 40%?"
+                checked={this.props.data.compensableVaServiceConnected}
+                required={!false}
+                onValueChange={(update) => {this.props.onStateChange('compensableVaServiceConnected', update);}}/>
+          }
 
-          <ErrorableCheckbox
-              label="Are you compensable VA Service Connected 0% - 40%?"
-              checked={this.props.data.compensableVaServiceConnected}
-              onValueChange={(update) => {this.props.onStateChange('compensableVaServiceConnected', update);}}/>
+          {this.props.data.isVaServiceConnected === true &&
```

Comment:
When I tried to consolidate two components inside the same bracket block, the linter complained.